ledger/participant-integration-api/src/main/scala/platform/indexer/ha/HaCoordinator.scala

```diff
@@ -58,8 +58,8 @@ case class HaConfig(
     workerLockAcquireRetryMillis: Long = 500,
     workerLockAcquireMaxRetry: Long = 1000,
     mainLockCheckerPeriodMillis: Long = 1000,
-    indexerLockId: Int = 0x646d6c00, // note 0x646d6c equals ASCII encoded "dml"
-    indexerWorkerLockId: Int = 0x646d6c01,
+    indexerLockId: Int = 0x646d6c0, // note 0x646d6c equals ASCII encoded "dml"
```

Comment:
FYI @meiersi-da (AFAIK you had the original idea for the default lock IDs) - the previous lock ids are too long for Oracle.

Reply:
Ack. Thanks for catching this.